lib/features/composer/presentation/composer_controller.dart

```diff
@@ -744,11 +751,8 @@ class ComposerController extends BaseController with DragDropFileMixin implement
       listFromIdentities.value = listIdentitiesMayDeleted;
 
       if (identitySelected.value == null) {
-        final selectedIdentityFromArguments = composerArguments.value?.selectedIdentity;
-        if (selectedIdentityFromArguments != null
-            && listFromIdentities.contains(selectedIdentityFromArguments)
-        ) {
-          await _selectIdentity(selectedIdentityFromArguments);
+        if (_selectedIdentityFromId(composerArguments.value?.selectedIdentityId) != null) {
```

Comment:
please use assignment to avoid duplicate  `_selectedIdentityFromId(composerArguments.value?.selectedIdentityId)`

Reply:
Done